exopy_hqc_legacy/instruments/drivers/dll/sp_adq14.py

```diff
@@ -99,6 +104,7 @@ def open_connection(self):
 
         """
         cu = ADQControlUnit(self._dll)
+        cu.enable_logging(3, b"C:\\Logs")
```

Comment:
This was just a quick hack I wrote to debug the issues I was having. It can be safely removed from this patch. (And added cleanly in the future)

Reply:
what do you mean added cleanly ? is this needed or not ?

Reply:
This is not needed, I added this to debug an issue with the sp_adq14. What I meant by added cleanly was the fact that we could have a settings somewhere to toggle logging on and off where we could choose the logging directory instead of hard coding all of that.